### PR TITLE
TR-105: Change the sidebar and top bar depending if the PLUGIN or the WEBAPP

### PR DESCRIPTION
**Pull request by binaryberserker**

### CODE REVIEW

File: pages/index.html.js [hunk 1]
```diff
@@ -2,68 +2,78 @@ const getSafeEnv = require("../server/client-env");
...
+        gtag('config', 'G-XQR3T6BZL3');
+      </script>
+    </head>
 
-	</body>
-	</html>
-	`;
+    <body class='overflow-x-hidden'>
+
+      ${mainFileRoute.includes("connect") ? '' : `
```
Comment (binaryberserker) [inline pages/index.html.js:37]: here's the primary change for the top header, we were actually already doing this 'includes("connect")' check, I just moved it up the DOM made it encapsulate the whole header

File: pages/index.html.js
Comment (binaryberserker) [inline pages/index.html.js:1]: made some formatting changes in index too cuz it was kind of all over the place

File: public/shared/main-helper.js [hunk 2]
```diff
@@ -65,6 +65,7 @@ export default async function mainHelper(config, { host, createStorage }) {
         loginComponent,
         mode: "TEAMS",
         storage,
+        isPlugin: host === 'jira'
```
Comment (binaryberserker) [inline public/shared/main-helper.js:68]: and passing is plugin for the configuration pane to use

File: public/timeline-configuration/timeline-configuration.js [hunk 3]
```diff
@@ -50,6 +50,26 @@ export class TimelineConfiguration extends StacheElement {
...
+                </a>
+              </div>
+              <div class="bitovi-poppins text-neutral-100 text-sm">
+                <a href="https://www.bitovi.com/services/agile-project-management-consulting" target="_blank">
+                  by Bitovi
+                </a>
+              </div>
+            </div>
+          </div>
+        {{/ if}}
```
Comment (binaryberserker) [inline public/timeline-configuration/timeline-configuration.js:71]: and the configuration pane header if running as a plugin

**Review by DavidNic11 — APPROVED** [review]

**Pull request merged**